tutorials/understanding-gov/index.md

```diff
@@ -0,0 +1,344 @@
+
+<HighlightBox type="note">
+
+In fact this file is already pinned on IPFS. Its CID is `QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49`. You can verify its content on <https://ipfs.io/ipfs/QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49>.
+
+</HighlightBox>
+
+Now look at the content of the generated `draft_proposal.json`:
+
+```json
+{
+  "metadata": "ipfs://CID",
+  "deposit": "10stake"
+}
+```
+
+Replace the `metadata` field with `ipfs://QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49`.
+
+Submit the proposal on chain from alice:
+
+```sh
+$ simd tx gov submit-proposal draft_proposal.json --from alice --keyring-backend test
+```
+
+The command outputs a transaction hash. You can use it to query the proposal:
+
+```sh
+$ simd query tx D8F1165AAB343EB9416F1DF3D30F2883D26E1125AED733878C590E60256ED9C9
+```
+
+## View and vote on proposals
+
+In your case, the proposal ID is `1`. You can query the proposal with the following command:
+
+```sh
+$ simd query gov proposal 1
+```
+
+Which returns:
+
+```sh
+deposit_end_time: "2022-09-20T16:36:04.741427768Z"
+final_tally_result:
+  abstain_count: "0"
+  no_count: "0"
+  no_with_veto_count: "0"
+  yes_count: "0"
+id: "1"
+messages: []
+metadata: ipfs://QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49
+status: PROPOSAL_STATUS_DEPOSIT_PERIOD
+submit_time: "2022-09-18T16:36:04.741427768Z"
+total_deposit:
+- amount: "10"
+  denom: stake
+voting_end_time: null
+voting_start_time: null
+```
+
+As you can see, the proposal is in the deposit period. This means that the deposit associated with it has not yet reached the minimum required, so you cannot vote on it just yet. Find out what is the minimum proposal deposit for a chain with the following command:
+
+
+```sh
+$ simd query gov params --output json | jq .deposit_params.min_deposit
+```
+
+It returns:
+
+```json
+[
+  {
+    "denom": "stake",
+    "amount": "10000000"
+  }
+]
+```
+
+Therefore, since you submitted the proposal with `10stake`, you need to top up the deposit with `9999990stake`. You can do so with Bob and the following command:
+
+```sh
+$ simd tx gov deposit 1 9999990stake --from bob --keyring-backend test
+```
+
+The proposal is now in the voting period. Do not forget, you have three minutes (`180s` as per the gov parameters) to vote on the proposal.
+
+```sh
+$ simd query gov proposal 1 --output json | jq .status
+```
+
+You can vote on it with the following command:
+
+```sh
+$ simd tx gov vote 1 yes --from alice --keyring-backend test
+$ simd tx gov vote 1 no --from bob --keyring-backend test
+```
+
+After waiting for the voting period, you can see that the proposal has passed.
+
+```sh
+$ simd query gov proposal 1 --output json | jq .status
+```
+
+This is because the governance proposal weights each vote by the amount of tokens staked. Alice owns staked tokens, while Bob has no staked tokens. So Bob's vote was not taken into consideration in the tally of the result.
```

Comment:
```suggestion
This is because the governance proposal weights each vote by the amount of tokens staked. Alice owns staked tokens, while Bob has no staked tokens at the end of the voting period. So Bob's vote was not taken into consideration in the tally of the result.
```